.Rbuildignore

```diff
@@ -26,3 +26,4 @@
 ^_pkgdown\.yml$
 ^docs$
 ^pkgdown$
+^data-raw$
```

Comment:
I am quite fond of this solution, it is leaner and nicely combines defaults and ranges on 1 look.
Just some thoughts:

- we should still reconsider how we present the numeric ranges, as it led to, understandable, confusion yesterday. Thus we might either want to move from ranges to discrete numeric steps ( e.g c(0, 0.1, 0.2, 0.3 0.4, 0.5) instead of c(0,0.5) or we should add min and max cols to the tibble and use those for numeric vars. Let us talk about that in the synch later.
- is there is an automated solution to make sure new sysdata is generated wehen data-raw/st_arguments is changed? If not it looks like an expected source of error
- we should get rid of all existing lookups then, but if I understand correctly you plan to do that anyway